_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -22740,12 +22740,14 @@
 "bpu" = (
 /obj/structure/table/reinforced,
 /obj/item/book/manual/wiki/sop_command,
-/obj/item/aicard,
-/obj/item/multitool/command,
 /obj/machinery/keycard_auth{
 	pixel_x = -6;
 	pixel_y = 8
 	},
+/obj/item/aicard{
+	pixel_x = null
```

Comment:
Weird varedit happened here

Reply:
Yeah I don't know either what happened but I fixed it